src/protean/core/entity.py

```diff
@@ -11,71 +11,100 @@
 
 logger = logging.getLogger('protean.core.entity')
 
+ENTITY_META_ATTRIBUTE_NAMES = ('ordering')
+
 
 class EntityBase(type):
     """
     This base metaclass sets a `_meta` attribute on the Entity to an instance
     of Meta defined in the Entity
 
-    It also sets dictionary named `declared_fields` on the `meta_` attribute.
+    It also sets dictionary named `declared_fields` on the `_meta` attribute.
```

Comment:
we cannot do this, as this makes `_meta` a private member and thus inaccessible for use outside

Reply:
Yeah, need to do one more thing here of exposing meta as `opts` or something similar, like Django does. I can do that next in a different PR.

This was optional, but I realized a full-blown `EntityMeta` implementation is necessary to be able to implement linked fields. Which is necessary to support hierarchy in entities and attribute overriding.

Reply:
Hmm okay please expose it then